db/migrate/20250723021552_add_gender_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenderToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :gender, :string
```

Comment:
```suggestion
    add_column :users, :gender, :integer
```
Nếu dùng enum thì kiểu integer mới đúng e nha